apps/backend/src/common/entities/FixedWidthRecord.ts

```diff
@@ -61,23 +59,26 @@ export class FixedWidthRecord<T extends IFixedWidthRecord<T>>
       const value = line
         .substring(options.start, options.start + options.width)
         .trim();
-      (target as any)[field] = value;
-      if (options.format) {
-        if (options.format.type === DataType.Integer) {
+
+      switch (options?.format?.type) {
+        case DataType.Integer:
           (target as any)[field] = parseInt(value);
-        } else if (options.format.type === DataType.Float) {
-          (target as any)[field] = new Decimal(value)
-            .toDecimalPlaces(2)
-            .toNumber();
-        } else if (options.format.type === DataType.Date) {
+          break;
+        case DataType.Decimal:
```

Comment:
So our old if/else was handling Float and not Decimal and the decimal.js docs are not helping me, will a Float pass this condition if passed in or does this effectively change behaviour?

Reply:
Float was not actually being used anywhere so I removed it for clean up.